apps/indexer/src/chain/chainSync.ts

```diff
@@ -25,6 +26,8 @@ import {
 import { nodeAccessor } from "./nodeAccessor";
 import { statsProcessor } from "./statsProcessor";
 
+const logger = LoggerService.forContext("apps/indexer/chain/ChainSync.ts");
```

Comment:
in the backend it's a bit different. We will consume these in some kind of logging tool like grafana and we have a standard approach to naming. let it be something like a uniqe name in CamelCase. In this case 
```suggestion
const logger = LoggerService.forContext("ChainSync");
```

In case it's a class, it would be a class name.